ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -2048,7 +2049,8 @@
       },
       {
         "UserRoles": [
-          "[RESPONDENTSOLICITORONE]"
+          "[RESPONDENTSOLICITORONE]",
```

Comment:
Does solicitor1 need access to respondent2LitigationFriend

Reply:
During a 1v2 same solicitor scenario the solicitor can enter in litigation details for either respondent1, respondent2 or both. So they need write access to respondent2 in that situation.